frontend/javascripts/admin/dataset/dataset_url_import.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { useState } from "react";
+import DatasetAddRemoteView from "admin/dataset/dataset_add_remote_view";
+import { useFetch } from "libs/react_helpers";
+import { APIDataStore } from "types/api_flow_types";
+import { getDatastores } from "admin/admin_rest_api";
+import { DatasetAddType, getPostUploadModal } from "./dataset_add_view";
+import * as Utils from "libs/utils";
+import _ from "lodash";
+import { useHistory } from "react-router-dom";
+
+export function DatasetURLImport() {
+  const history = useHistory();
+  const [hasFetched, setHasFetched] = useState(false);
+  const [datastores, setDatastores] = useState<APIDataStore[]>([]);
+  const [datasetName, setDatasetName] = useState("");
+  const [organization, setOrganization] = useState("");
+  const [datasetNeedsConversion, setDatasetNeedsConversion] = useState(false);
+  const [datasetAddType, setImportType] = useState<DatasetAddType>(DatasetAddType.UPLOAD);
+  const [showPostUploadModal, setShowPostUploadModal] = useState(false);
+  useFetch(
+    async () => {
+      setDatastores(await getDatastores());
+      setHasFetched(true);
+    },
+    null,
+    [],
+  );
+  const params = Utils.getUrlParamsObject();
+  const datasetUri = _.has(params, "url") ? params.url : null;
+  const handleDatasetAdded = async (
+    datasetAddType: DatasetAddType,
+    datasetOrganization: string,
+    uploadedDatasetName: string,
+    needsConversion: boolean | null | undefined,
+  ): Promise<void> => {
+    setOrganization(datasetOrganization);
+    setDatasetName(uploadedDatasetName);
+    setImportType(datasetAddType);
+    if (needsConversion != null) setDatasetNeedsConversion(needsConversion);
+    setShowPostUploadModal(true);
+  };
+
+  const postUploadModal = () => {
+    if (!showPostUploadModal) return null;
+    if (!datasetNeedsConversion) {
+      history.push(`/datasets/${organization}/${datasetName}/view`);
```

Comment:
oh, good point. I think I didnt test it yet because I wasnt sure which DS would need conversion. I will find that out. 
I was assuming this could happen as there is a `needsConversion` prop in `dataset_add_remote_view`

Reply:
I think, the parameter is only there so that the interface matches the other import methods (i.e., upload dataset). I had a quick look and it looks like the "import remote" view never passes true for that parameter. I think it's also valid to assume that this will stay that way because otherwise one would need to convert a DS that is only remotely available and that should be streamed.